### PR TITLE
Live pics

### PR DESCRIPTION
**Pull request by AndrewHarper041**

### CODE REVIEW

File: source/PHP/uploadPhoto.php [hunk 1]
```diff
@@ -1,26 +0,0 @@
-<?php
```
Comment (domgrigore) [inline source/PHP/uploadPhoto.php]: Why was this file deleted?

Comment (graphitezeppelin) [inline source/PHP/uploadPhoto.php]: I changed uploadPhoto to uploadPic for consistent naming convention
